src/AM.Condo/Targets/Publish/Go.targets

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="GoBuild" Condition="$(GoBuild)">
+      <PropertyGroup>
+        <GoBuildOptions Condition=" '$(GoBuildOptions)' == '' ">$(GO_BUILD_OPTIONS)</GoBuildOptions>
+        <GoBuildOptions Condition=" '$(GoBuildOptions)' == '' ">-buildmode=$(GoBuildMode)</GoBuildOptions>
```

Comment:
should this always be appended to GoBuildOptions -- even if a GO_BUILD_OPTIONS are defined? What happens when someone defines both GO_BUILD_OPTIONS and GoBuildMode, for instance?

Reply:
I was wrestling with this myself. it might be better to just remove GoBuildMode, the default build mode for go is exe anyway. if someone wants to change how publish works then it should only be via the GO_BUILD_OPTIONS. what do you think?

Reply:
works for me!